packages/main/src/components/AnalyticalTable/index.tsx

```diff
@@ -252,6 +252,12 @@ const AnalyticalTable: FC<TableProps> = forwardRef((props: TableProps, ref: Ref<
     useRTL
   );
 
+  // scroll bar detection
+  useEffect(() => {
+    const visibleRowCount = rows.length < visibleRows ? Math.max(rows.length, minRows) : visibleRows;
+    dispatch({ type: 'TABLE_SCROLLING_ENABLED', payload: { isScrollable: data.length > visibleRowCount } });
+  }, [rows.length, minRows, visibleRows]);
```

Comment:
Add `data.length` to dependencies array
```suggestion
  }, [rows.length, minRows, visibleRows, data.length]);
```

Reply:
changed it to use rows.length - otherwise filtering would not be respected